ratpack-core/src/main/java/ratpack/http/Status.java

```diff
@@ -25,16 +25,86 @@
 public interface Status {
 
   /**
-   * The 200 status code.
+   * The 1xx status codes.
+   */
+
+  Status CONTINUE = Status.of(100);
+  Status SWITCHING_PROTOCOLS = Status.of(101);
+  Status PROCESSING = Status.of(102);
+  Status EARLY_HINTS = Status.of(103);
+
+  /**
+   * The 2xx status codes.
    */
   Status OK = Status.of(200);
+  Status CREATED = Status.of(201);
+  Status ACCEPTED = Status.of(202);
+  Status NON_AUTHORITATIVE_INFO = Status.of(203);
+  Status NO_CONTENT = Status.of(204);
+  Status RESET_CONTENT = Status.of(205);
+  Status PARTIAL_CONTENT = Status.of(206);
+  Status MULTI_STATUS = Status.of(207);
+  Status ALREADY_REPORTED = Status.of(208);
+  Status IM_USED = Status.of(226);
 
   /**
-   * The 304 status code.
-   *
-   * @since 1.4
```

Comment:
We need to keep the javadoc including the `@since` because this is public API. Each `enum` added here needs to have it's own javadoc comment declaring it is available since 1.6

Reply:
Okay, wasn't sure how you guys would want to handle that and figured I'd get feedback on it during the PR. I'll push an update in about 10 minutes.

Reply:
No problem. Yeah, any public API needs javadoc declaring the version it became available. Thanks!